examples/1App_ACCERT_PWR12BE/watts_exec.py

```diff
@@ -0,0 +1,30 @@
+
+"""
+This example demonstrates how to use WATTS to perform 
+ACCERT simulations. The ACCERT model quotes the total cost 
+and the cost of each account for a given set of parameters. 
+Results from ACCERT are extracted and stored in params.
+"""
+
+import watts
+
+# Set required parameters
+params = watts.Parameters()
+params['thermal_power'] = 3200
+params['electric_power'] = 1300
+params['cost_217'] = 29000000
+input_name = "ACCERT_input.tmpl"
+
+
+accert_plugin = watts.PluginACCERT(f'{input_name}')
+accert_result = accert_plugin(params)
+
+print(accert_result.inputs)
+print(accert_result.outputs)
+print(accert_result.total_cost)
+
+# ### uncomment below to see the ACCERT account table
+# print(accert_result.account_table.to_markdown())
```

Comment:
this currently doesn't work for me - I have the following error - are we missing some utility in requirements? 
```
Traceback (most recent call last):
  File "watts_exec.py", line 27, in <module>
    print(accert_result.account_table.to_markdown())
  File "/Users/nstauff/opt/anaconda3/lib/python3.7/site-packages/pandas/core/generic.py", line 5179, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'to_markdown' 
```

Reply:
pandas should have to_markdown function after version 1.0.0, I guess it might be the version of pandas is below 1.0.0

Reply:
should I add a pandas version in pyproject.toml?

Reply:
I am not sure - I don't know how to force update on installation while it says I am meeting requirements with current version. Maybe @paulromano can help give directions on how we can change WATTS requirements to specify pandas version installed. 

```
(base) c356021:1App_ACCERT_PWR12BE nstauff$ pip3 install pandas
Requirement already satisfied: pandas in /Users/nstauff/opt/anaconda3/lib/python3.7/site-packages (0.25.1)
```

Reply:
Pandas might need an optional dependency for "tabulate", if you can test whether if you can install or update tabulate? If this work, I think I will add it to the project.toml

Reply:
Thanks @JiaZhou-PU  - this was resolved by running `pip install -U pandas-profiling`